storyblok/field-plugin
# fix(sandbox): Upgrading @storyblok/mui dependency

Issue: EXT-1644

## What?

Fixed transparent alert:

<img width="852" alt="image" src="https://github.com/storyblok/field-plugin/assets/14206504/c72ab0b2-dfc1-4dac-97cf-96b242ecf380">

Updates the accordion style to be consistent with blok.ink:

<img width="796" alt="image" src="https://github.com/storyblok/field-plugin/assets/14206504/11d78de9-b844-403b-9b01-63d0bcc1ca1a">

## How to test? (optional)

See the preview deployment.

It should be possible to expand/collapse the accordions and to resize the preview area as before:

![2023-05-29_15-45-35 (1)](https://github.com/storyblok/field-plugin/assets/14206504/82b33c87-34e2-4e4d-8bb5-975bb19bccdc)
